CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+We encourage you to contriubte to this project in any way you can! An important goal of this project is to foster a community around these tools. If you'd like to be a part of that community, consider contributing in one of these ways:
+
+## Discussion
+
+Discussion gives us invaluable information about the project's requirements and goals. If you would like to contribute to the ongoing discussion of the project's direction and future goals, join the conversation on the relevant [Discussion Issue](discussionIssues), create a [new Issue](newIssue) for your discussion topic, or strike up a conversation in the project's [Gitter chat room](gitter).
```

Comment:
I think you need to change the `[Discussion Issue](discussionIssues)` to `[Discussion Issue][discussionIssues]`(square brackets) for the markdown variables to work. Or change it to `[Discussion Issue](https://github.com/mozillascience/software-citation-tools/issues?q=is%3Aissue+is%3Aopen+label%3ADiscussion)`